autogluon/inference/docker/0.8/py3/Dockerfile.cpu.os_scan_allowlist.json

```diff
@@ -0,0 +1,60 @@
+{
+  "grpcio": [
```

Comment:
`reason_to_ignore` is missing for each item. See example [here](https://github.com/aws/deep-learning-containers/blob/master/autogluon/training/docker/0.6/py3/Dockerfile.cpu.os_scan_allowlist.json#L30). Must be last entry in a block.

```
      "reason_to_ignore": "patch via AWS binaries in parent container"
```